### PR TITLE
Add README and CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by egegunes**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
+# Contributing to the Operator
+
+Percona welcomes and encourages community contributions to help improve [Percona Kubernetes Operator for Percona Server for MySQL](https://www.percona.com/doc/kubernetes-operator-for-pxc/index.html). The Operator automates the creation, modification, or deletion of items in your Percona Server for MySQL environment, and contains the necessary Kubernetes settings to maintain a consistent Percona Server for MySQL instance.
```
Comment (egegunes) [inline CONTRIBUTING.md:3]: Fix doc link

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+### 2. Code of Conduct
+
+Please make sure to read and observe the [Contribution Policy](code-of-conduct.md).
+
+## Submitting a pull request
+
+Improvement and bugfix tasks for the Operator are tracked in [Jira](https://jira.percona.com/projects/K8SPS/issues). Although not mandatory, it is a good practice to examine already open Jira issues before submitting a pull request. For bigger contributions, we suggest creating a Jira issue first and discussing it with the engineering team and community before proposing any code changes.
+
+Another good place to discuss Percona Kubernetes Operator for Percona Server for MySQL with developers and other community members is the [community forum](https://forums.percona.com/categories/kubernetes-operator-percona-xtradb-cluster).
```
Comment (egegunes) [inline CONTRIBUTING.md:21]: Fix forum link

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+1. First, you need to [fork the repository on GitHub](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/syncing-a-fork), clone your fork locally, and then [sync your local fork to upstream](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/syncing-a-fork). After that, before starting to work on changes, make sure to always sync your fork with upstream. 
+2. Create a branch for changes you are planning to make. If there is a Jira ticket related to your contribution, it is recommended to name your branch in the following way: `<Jira issue number>-<short description>`, where the issue number is something like `K8SPS-622`.
+
+   Create the branch in your local repo as follows:
+
+   ```
+   git checkout -b K8SPS-622-fix-feature-X
+   ```
+
+   When your changes are [ready](e2e-tests/README.md), make a commit, mentioning the Jira issue in the commit message, if any:
```
Comment (egegunes) [inline CONTRIBUTING.md:36]: Provide README for `e2e-tests`

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+4. When the reviewer makes some comments, address any feedback that comes and update the pull request.
+5. When your contribution is accepted, your pull request will be approved and merged to the main branch.
+
+
+### 2. Contributing to documentation
+
+The workflow for documentation is similar. Please take into account a few things:
+
+1. All documentation is written using the [Sphinx engine markup language](https://www.sphinx-doc.org/). Sphinx allows easy publishing of various output formats such as HTML, LaTeX (for PDF), ePub, Texinfo, etc.
+2. We store the documentation as *.rst files in the [pxc-docs](https://github.com/percona/percona-xtradb-cluster-operator/tree/pxc-docs) branch of the Operator GitHub repository. The documentation is licensed under the [Attribution 4.0 International license (CC BY 4.0)](https://creativecommons.org/licenses/by/4.0/).
```
Comment (egegunes) [inline CONTRIBUTING.md:54]: Fix repo and branch link

File: README.md [hunk 5]
```diff
@@ -1,19 +1,50 @@
...
-## Build
+[![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
 
-```
-$ make build
-```
+[Percona Server for MySQL](https://www.percona.com/software/mysql-database/percona-server) is a free, fully compatible, enhanced, and open source drop-in replacement for any MySQL database. It provides superior performance, scalability, and instrumentation.
 
-## Install CRDs
+Based on our best practices for deployment and configuration, [Percona Kubernetes Operator for Percona Server for MySQL](https://www.percona.com/doc/kubernetes-operator-for-pxc/index.html) contains everything you need to quickly and consistently deploy and scale Percona Server for MySQL instances in a Kubernetes-based environment on-premises or in the cloud. It provides the following capabilities:
```
Comment (egegunes) [inline README.md:7]: Fix doc link

File: README.md [hunk 6]
```diff
@@ -1,19 +1,50 @@
...
+Percona Operators are based on the [Operator SDK](https://github.com/operator-framework/operator-sdk) and leverage Kubernetes primitives to follow best CNCF practices.
+
+## Installation
+
+It usually takes two steps to deploy Percona Server for MySQL on Kubernetes:
+
+* Deploy the operator from `deploy/bundle.yaml`
+* Deploy the database cluster itself from `deploy/cr.yaml`
+
+See full documentation with examples and various advanced cases on [percona.com](https://www.percona.com/doc/kubernetes-operator-for-pxc/index.html).
```
Comment (egegunes) [inline README.md:39]: Fix doc link

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| init-deploy | passed |
| semi-sync | passed |
| monitoring | passed |
| sidecars | passed |

commit: https://github.com/percona/percona-server-mysql-operator/pull/28/commits/42cb16ff6c161427d511c8ea6cacb9bd725da4ba
image: `perconalab/percona-server-mysql-operator:PR-28-42cb16f`

File: README.md [hunk 7]
```diff
@@ -1,19 +1,50 @@
-# Percona Distribution for MySQL Operator
+# Percona Kubernetes Operator for Percona Server for MySQL
```
Comment (spron-in) [inline README.md:1]:
```suggestion
# Percona Distribution for MySQL Operator v2
```

Comment (spron-in) [reply README.md:1]: that is the official name.

File: README.md (hunk 5 above)
Comment (spron-in) [inline README.md:7]:
```suggestion
Based on our best practices for deployment and configuration, [Percona Distribution for MySQL Operator v2](https://www.percona.com/doc/kubernetes-operator-for-pxc/index.html) contains everything you need to quickly and consistently deploy and scale Percona Server for MySQL instances in a Kubernetes-based environment on-premises or in the cloud. It provides the following capabilities:
```

Comment (spron-in) [reply README.md:7]: @fiowro need to change the link

File: README.md [hunk 8]
```diff
@@ -1,19 +1,50 @@
...
+It usually takes two steps to deploy Percona Server for MySQL on Kubernetes:
+
+* Deploy the operator from `deploy/bundle.yaml`
+* Deploy the database cluster itself from `deploy/cr.yaml`
+
+See full documentation with examples and various advanced cases on [percona.com](https://www.percona.com/doc/kubernetes-operator-for-pxc/index.html).
+
+## Contributing
+
+Percona welcomes and encourages community contributions to help improve Percona Kubernetes Operator for Percona Server for MySQL.
```
Comment (spron-in) [inline README.md:43]:
```suggestion
Percona welcomes and encourages community contributions to help improve open source software.
```

File: CONTRIBUTING.md [hunk 9]
```diff
@@ -0,0 +1,67 @@
...
+1. All documentation is written using the [Sphinx engine markup language](https://www.sphinx-doc.org/). Sphinx allows easy publishing of various output formats such as HTML, LaTeX (for PDF), ePub, Texinfo, etc.
+2. We store the documentation as *.rst files in the [ps-docs](https://github.com/percona/percona-server-mysql-operator/tree/ps-docs) branch of the Operator GitHub repository. The documentation is licensed under the [Attribution 4.0 International license (CC BY 4.0)](https://creativecommons.org/licenses/by/4.0/).
+
+After [installing Sphinx](https://www.sphinx-doc.org/en/master/usage/installation.html) you can use `make html` or `make latexpdf` commands having the documentation branch as your current directory to build HTML and PDF versions of the documentation respectively.
+
+## Code review
+
+### 1. Automated code review
+
+Your pull request will go through an automated build and testing process, and you will have a comment with the report once all tests are over (usually, it takes about 3 hours).
```
Comment (hors) [inline CONTRIBUTING.md:62]: Now it takes 1 hour :)
```suggestion
Your pull request will go through an automated build and testing process, and you will have a comment with the report once all tests are over (usually, it takes about 1 hour).
```

File: CONTRIBUTING.md [hunk 10]
```diff
@@ -0,0 +1,67 @@
...
+
+### 2. Code review by the Operator developers
+
+Your contribution will be reviewed by other developers contributing to the project. The more complex your changes are, the more experts will be involved. You will receive feedback and recommendations directly on your pull request on GitHub, so keep an eye on your submission and be prepared to make further amendments. The developers might even provide some concrete suggestions on how to modify your code to better match the project’s expectations.
+
```
Comment (hors) [inline CONTRIBUTING.md:67]: Also we need to add 'Building and automated testing' section. But we can do it via separate PR in next release

https://github.com/percona/percona-xtradb-cluster-operator/blob/main/CONTRIBUTING.md#building-and-automated-testing

File: README.md [hunk 11]
```diff
@@ -1,19 +1,50 @@
...
+* Deploy the operator from `deploy/bundle.yaml`
+* Deploy the database cluster itself from `deploy/cr.yaml`
+
+See full documentation with examples and various advanced cases on [percona.com](https://www.percona.com/doc/kubernetes-operator-for-mysql/index.html).
+
+## Contributing
+
+Percona welcomes and encourages community contributions to help improve Percona Distribution for MySQL Operator.
+
+See the [Contribution Guide](CONTRIBUTING.md) and [Building and Testing Guide](e2e-tests/README.md) for more information.
```
Comment (hors) [inline README.md:45]: [Building and Testing Guide](e2e-tests/README.md) - we do not have it now

**Review by hors — CHANGES_REQUESTED** [review]

**Review by hors — APPROVED** [review]

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| init-deploy | passed |
| semi-sync | passed |
| monitoring | passed |
| sidecars | passed |

commit: https://github.com/percona/percona-server-mysql-operator/pull/28/commits/18704b9fcf3433e68012924606ee4a5b47b7dd4b
image: `perconalab/percona-server-mysql-operator:PR-28-18704b9`

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| init-deploy | passed |
| semi-sync | passed |
| monitoring | passed |
| sidecars | passed |

commit: https://github.com/percona/percona-server-mysql-operator/pull/28/commits/6c8d6b9d7b803edf45a110809d53c9bb073d52dd
image: `perconalab/percona-server-mysql-operator:PR-28-6c8d6b9`

**Review by fiowro — APPROVED** [review]
LGTM

**Pull request merged**